src/page-multi-facility/FacilityRowRtValuePill.tsx

```diff
@@ -0,0 +1,62 @@
+import React from "react";
+import styled from "styled-components";
+
+import { RateOfSpreadType } from "../constants/EpidemicModel";
+import Colors from "../design-system/Colors";
+import PillCircle from "../design-system/PillCircle";
+
+const pillColors: {
+  [x: string]: {
+    [x: string]: string;
+  };
```

Comment:
would something more specific like 
```
{
  [key in RateofSpreadType]: {
    text: string;
    border: string;
  };
}
```
work here instead? (probably not exactly because I am bad at TypeScript but hopefully you get the general idea)

Reply:
Your typescript > my typescript.

That looks good. I had to change RateOfSpreadType to an enum, but this is better.

Reply:
it doesn't look like anything actually was changed here?

Reply:
Odd. I changed it but may have accidentally reverted through merging with master.